packages/itwin/reports-config-widget/src/widget/components/HorizontalTile.tsx

```diff
@@ -6,51 +6,47 @@ import type { ReactNode } from "react";
 import React from "react";
 import { Text } from "@itwin/itwinui-react";
 import "./HorizontalTile.scss";
+import classNames from "classnames";
 
-interface HorizontalTileProps {
+export interface HorizontalTileProps {
```

Comment:
I'm not liking the any's. Can we extend this with ```React.ComponentPropsWithoutRef<"div">``` and in the container spread the props ```{...props}``` and remove the onClick. Also ```onClickTitle``` type can be ```React.MouseEventHandler```

Reply:
I don't think it would be as simple as spreading the props in that case. Caller could override the classNames, which would break the ui.

I agree that `any`s could be changed to some types.

Reply:
Also, do the `onClick` and `onClickTitle` even need parameters? Maybe we could get rid of them?

Reply:
className={classNames(
       'rcw-horizontal-tile-container',
        { "rcw-horizontal-tile-selected": props.selected },
        props.className
      )}`
This would stop it from being overridden and would just extend it.

Reply:
`onClick` event is now extended with `React.ComponentPropsWithoutRef<"div">`. Props are now extended.